adding symphony image build files to symphony-sample

abbasally5 opened the pull request.

Commit:
adding symphony image build files to symphony-sample

riethm commented on symphony-sample/packer-templates/scripts/playbook.yml:
```diff
@@ -0,0 +1,107 @@
+#  - name: Copy over symphony playbook
+#    copy:
+#      src: /root/symphony-automation
+#      dest: /tmp
```
Remove unused commented code blocks

abbasally5 replied:
removed in latest push. thanks for catching it

riethm commented on symphony-sample/packer-templates/scripts/playbook.yml:
```diff
@@ -0,0 +1,107 @@
+  - name: Copy over symphony systemd script
+    template:
+      src: symphony.j2
+      dest: /usr/bin/symphony
```
We probably should put this somewhere else, or rename it to something less likely to be invoked by accident

abbasally5 replied:
renamed to symphony-install. I also renamed the log file to /var/log/symphony-install.txt. Should this be updated in the tutorial?

riethm commented on symphony-sample/packer-templates/scripts/playbook.yml:
```diff
@@ -0,0 +1,107 @@
+      remote_src: yes
+
+  - name : Only save symphony-automation playbook
+    shell: mv /tmp/hpc-cluster-master/symphony-automation /tmp && rm -rf /tmp/hpc-cluster-master /tmp/hpc-cluster-master.zip
```
Update the source path (`symphony-sample/automation`)

abbasally5 replied:
updated in latest push

riethm commented on symphony-sample/packer-templates/scripts/files/extras.repo:
```diff
@@ -0,0 +1,4 @@
+[Extras]
+name=extras
+baseurl="http://mirrors.service.networklayer.com/centos/7/extras/x86_64/"
```
This would only work if the build environment is an IBMCloud hosted machine

abbasally5 replied:
changed to http://mirror.centos.org/centos/7/extras/x86_64/. Thanks, didn't think about this.

riethm commented on symphony-sample/packer-templates/scripts/files/os.repo:
```diff
@@ -0,0 +1,4 @@
+[OS]
+name=os
+baseurl="http://mirrors.service.networklayer.com/centos/7/os/x86_64/"
```
This would only work if the build environment is an IBMCloud hosted machine

abbasally5 replied:
changed to http://mirror.centos.org/centos/7/os/x86_64/

riethm commented on symphony-sample/packer-templates/scripts/files/updates.repo:
```diff
@@ -0,0 +1,4 @@
+[Updates]
+name=updates
+baseurl="http://mirrors.service.networklayer.com/centos/7/updates/x86_64/"
```
This would only work if the build environment is an IBMCloud hosted machine

abbasally5 replied:
changed to http://mirror.centos.org/centos/7/updates/x86_64/

riethm commented on symphony-sample/packer-templates/scripts/inventory:
```diff
@@ -0,0 +1,4 @@
+[all:vars]
+ansible_connection=ssh
+ansible_user=root
+ansible_ssh_pass=p@ssw0rd
```
Can we provide the ssh_pass through the command line, and have it use the packer variable? Otherwise, it's one more place the user has to update, to use a non-default pw

abbasally5 replied:
Removed the inventory file. ansible now uses the ssh_username and ssh_pass from the packer builder step

Commit:
Getting symphony-automation playbook from symphony-sample directory now
- Renaming symphony script to symphony-install
- Changing yum repo urls from private softlayer urls to centos urls
- Adding root_password user variable
- Removing inventory file

Commit:
using packer builder ssh_username and ssh_password for ansible ssh
